test/test_tsd_types.py

```diff
@@ -27,7 +27,7 @@
 from test.backend_test_util import _mock_output
 
 
-def _make_backend(target_folder_path, template_path, custom_args=None):
+def _make_backend(target_folder_path, template_path, custom_args=None):  # type: ignore
     # type: (typing.Text, typing.Text, typing.List) -> TSDTypesBackend
```

Comment:
Not for the sake of this PR, but we should be able to update these to modern type hint syntax right?